### DISCUSSION TITLE
Business support

### DISCUSSION THREAD

**Discussion by pabau** [prisma/prisma#8758] [Q&A] [answered]
We would like to inquiry about getting a premium support from you for which we are willing to pay. We are just starting using prisma and the git issues tracker is not the most convenient way, most of the issues we have might not even be prisma issues. Would you be able to provide us such service?

**Comment by chrismatteson** [comment, marked as answer]
I run sales for Prisma. I’d love to discuss this with you further. Please reach out to me at matteson@prisma.io to discuss.

Thank you.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
